contrib/local-environment/oauth2-proxy-traefik.cfg

```diff
@@ -0,0 +1,23 @@
+http_address="0.0.0.0:4180"
+cookie_secret="OQINaROshtE9TcZkNAm-5Zs2Pv3xaWytBmc5W7sPX7w="
+email_domains=["example.com"]
+cookie_secure="false"
+# upstreams="http://httpbin"
+upstreams="static://202"
+cookie_domains=[".localtest.me"] # Required so cookie can be read on all subdomains.
+whitelist_domains=[".localtest.me"] # Required to allow redirection back to original requested target.
+
+# keycloak provider
+client_secret="72341b6d-7065-4518-a0e4-50ee15025608"
+client_id="oauth2-proxy"
+redirect_url="http://oauth2-proxy.localtest.me/oauth2/callback"
+
+# in this case oauth2-proxy is going to visit
+# http://keycloak.localtest.me:9080/auth/realms/master/.well-known/openid-configuration for configuration
+oidc_issuer_url="http://keycloak.localtest.me/auth/realms/master"
+provider="oidc"
+provider_display_name="Keycloak"
+
+reverse_proxy="true"
+
+skip_provider_button="true"
```

Comment:
I think these are new right? Perhaps some comments why these are here would be good for future users

Reply:
Indeed this is possible thanks to the new documented `ForwardAuth` + static upstream configuration.

I added a comment.